src/net/sourceforge/kolmafia/KoLAdventure.java

```diff
@@ -888,7 +888,8 @@ public boolean canAdventure() {
           // Right Side of the Tracks and The Nearby Plains
         case TOWN, PLAINS -> true;
           // The Distant Woods, Forest Village, The 8-Bit Realm
-        case WOODS, VILLAGE, REALM -> QuestDatabase.isQuestStarted(Quest.LARVA);
+        case WOODS, VILLAGE, REALM -> QuestDatabase.isQuestStarted(Quest.LARVA)
+            || QuestDatabase.isQuestStarted(Quest.CITADEL);
```

Comment:
given this appears in a bunch of places (that you've just added), could we extract to a function like `isWoodsOpen` (that checks LARVA or CITADEL) and call that?